microsoft-365/compliance/sit-get-started-exact-data-match-test.md

```diff
@@ -79,6 +79,12 @@ If you don't find any matches, try the following:
 
 - If the SIT you selected for a primary element in the EDM type doesn't find a match in the item or finds fewer matches than you expected, check that it supports the separators and delimiters that are in the content. Be sure to include the ignored delimiters defined in your schema.
 
+- If the SIT you selected for a primary element in the EDM type does find matches in the item but those matches are not found when testing the EDM Sensitive information type itself, it may be that the sensitive information type you defined is starting or ending the match at the wrong point in the text. The two most common causes for this are:
```

Comment:
```suggestion
- If the SIT you selected for a primary element in the EDM type does find matches in the item but those matches are not found when testing the EDM sensitive information type itself, it may be that the sensitive information type you defined is starting or ending the match at the wrong point in the text. The two most common causes for this are:
```